oauth2_provider/oauth2_validators.py

```diff
@@ -572,7 +572,7 @@ def validate_user(self, username, password, client, request, *args, **kwargs):
         """
         Check username and password correspond to a valid and active User
         """
-        u = authenticate(username=username, password=password)
+        u = authenticate(request, username=username, password=password)
         if u is not None and u.is_active:
```

Comment:
`is_active` is already checked in django (>=1.11 I think) do we need to check this here again?

Reply:
This fix make sure request as a parameter to authenticate function, so authentication module can get request information.

Reply:
@cleder I agree regarding `is_active`, in fact Django also skips the check if user model doesn't implement `is_active`. Let's remove this part.

Reply:
It could be a separate PR though. Approving as the fix is correct.